chk/modules/http/request_helper.py

```diff
@@ -1,13 +1,15 @@
 """
 Http module helpers
 """
+from types import MappingProxyType
+from urllib.parse import unquote, urlparse
+
+from requests import request, Response
+from requests.auth import HTTPBasicAuth
+
 from chk.infrastructure.work import RequestProcessorContract
```

Comment:
- organize by alphabetic order
- remove unnecessary new line

```suggestion
from types import MappingProxyType
from urllib.parse import unquote, urlparse
from requests import request, Response
from requests.auth import HTTPBasicAuth
from chk.infrastructure.work import RequestProcessorContract
```

Reply:
According to the python [PEP8](https://peps.python.org/pep-0008/#imports) guideline, I think we should put a blank line between each group of imports. @0hsn

Reply:
fair enough